test/unit-test/scripts/npm/prepublish/lib/changelog-spec.js

```diff
@@ -0,0 +1,32 @@
+const path = require( 'path' );
+const util = require( '../../../../../../scripts/npm/prepublish/lib/' );
```

Comment:
This could be updated to be consistent with the other files.

```
 const rootPath = require( '../root-path' );
const libPath = path.join( rootPath, 'scripts', 'npm', 'prepublish', 'lib' );
const util = require( libPath );
```

Reply:
Fixed